pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -626,8 +626,9 @@ func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.Servic
 
 	// Check to see if Docker Compose v3 Deploy.Mode has been set to "global"
 	if service.DeployMode == "global" {
-		log.Warning("Global mode not yet supported, containers will only be replicated once throughout the cluster. DaemonSet support will be added in the future.")
-		replica = 1
+		opt.CreateD = false
+		opt.CreateDS = true
+		opt.Controller = "daemonset"
```

Comment:
At warn message is needed if the user specific a `--deployment` flag.Also in my opinion, it's better to use deployment if the user provide a `--deployment` flag. Because if we ignore it,  we can only create a daemonset if `deployMode==global`, there is no way to overwrite it. But on the other side, we can create daemonset(by default) and deployment(user provide deployment flag), I think this will give the user more choices.

Reply:
Sure, agree this opinion.  Change code to that if user use `--deployment` flag, it will convert to `Deployment` and default to use `StatefulSet`.   Also add test case to this situation. @hangyan